.github/workflows/build_and_upload_theme_on_push_to_dev.yaml

```diff
@@ -0,0 +1,69 @@
+# Jobs to build and deploy a Docker image for Fairspace Keycloak theme on push to the dev or release branches
+
+name: Build & Upload Fairspace Keycloak Theme Docker image
+
+env:
+  DOCKER_REGISTRY: ghcr.io
+
+on:
+  push:
+    branches:
+      - dev
+      - release
+
+jobs:
+  # A job to generate one shared unique version tag per build cycle for all built artifacts
+  generate-version:
+    runs-on: ubuntu-latest
+    outputs:
+      output1: ${{ steps.version.outputs.keycloak_fairspace_theme_version }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - id: version
+        name: Generating version tag for artifacts
```

Comment:
Iit could be just a step in build-and-upload-docker-image-for-keycloak-fairspace-theme job as there are no other jobs depending on that. As a result we would not need to checkout repo twice